data-scripts/tosdr.js

```diff
@@ -73,6 +73,32 @@ function getSitePoints (sites) {
             points.match.bad = points.all.bad
         }
 
+        if (points.class === 'A' || points.class === 'B' || points.score) {
+            let normalizedScore
```

Comment:
super super minor: could this be 'let normalizedScore = 5', so that we can remove the else statement below?

Reply:
Feel free to ignore heh :)